modal/app.py

```diff
@@ -824,9 +824,16 @@ async def spawn_sandbox(
             raise InvalidError("`app.spawn_sandbox` requires a running app.")
 
         if _allow_background_volume_commits is False:
-            deprecation_warning(
+            deprecation_error(
                 (2024, 5, 13),
-                "Disabling volume background commits is now deprecated. Set _allow_background_volume_commits=True.",
+                "Disabling volume background commits is now deprecated. "
+                "Remove this argument to enable the functionality.",
```

Comment:
Something about this reads a little confusingly to me — but not a huge deal.

Reply:
Yeh I tried a few wordings and couldn't come up with something nice.